plugins/wme-sitebuilder/wme-sitebuilder/Plugin.php

```diff
@@ -49,12 +56,62 @@ public function __construct( Container $container, LoggerInterface $logger ) {
 	 * @return self
 	 */
 	public function init() {
+		// Boot all service providers.
+		$this->bootServiceProviders();
+
+		// Register the all service providers' WordPress hooks.
+		add_action( 'plugins_loaded', function () {
+			$this->loadServiceProviders();
+		}, 0 );
+
 		// Load all registered modules.
 		$this->loadModules();
 
+		// Add admin color scheme.
+		$admin = new AdminColorScheme();
+		$admin->add_admin_color_scheme();
+		$admin->set_admin_color_scheme();
```

Comment:
This would constantly reset the user's color scheme to ours, bypassing the safeguard in [`set_color_scheme()`](https://github.com/moderntribe/wme/pull/197/files#diff-a99b1e325f72f64aa9b8dc2b0bb8a8565050e4236b1b6f2c5a6686c86514a879R69)